tests/unit/LanguageTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+class LanguageTest extends WikiaBaseTest {
+
+	/**
+	 * @dataProvider shortenNumberDecoratorDataProvider
+	 * @group UsingDB
+	 */
+	public function testShortenNumberDecorator($number,$expected) {
+		$this->mockMessage( 'number-shortening-billions', '$1B' );
+		$this->mockMessage( 'number-shortening-millions', '$1M' );
+		$this->mockMessage( 'number-shortening', '$1K' );
+
+		$lang = new Language();
+
+		$result = $lang->shortenNumberDecorator($number);
+		$this->assertEquals($expected, $result);
```

Comment:
Spacey per conventions, same in the data provider below (and those could probably be short array syntax as well).